tdrs-frontend/.env.test

```diff
@@ -1,3 +1,8 @@
+###
+# Test environment-specific settings
+# This file is loaded when running `npm test`
```

Comment:
Same here, a matter of preference but we could leave this file blank as well

Reply:
Or even delete it? (But — these are potential future simplifications, let's not let them hold up this PR!)